src/LibPQ.jl

```diff
@@ -7,15 +7,15 @@ using Compat.Dates
 using DocStringExtensions
 using Decimals
 using DataStreams
-using Base.Iterators: zip, product
-using IterTools: imap
+import Base.Iterators: zip, product
```

Comment:
Please use `using` and not `import` for all of these, including Memento (although the explicit imports are fine)